src/routes/console/organization-[organization]/+layout.ts

```diff
@@ -49,4 +56,5 @@ export const load: LayoutLoad = async ({ params, depends }) => {
         sdk.forConsole.account.updatePrefs(newPrefs);
         error(e.code, e.message);
     }
+    return {roles, scopes};
```

Comment:
this is dead code, since the `error()` function in the catch throws an error for error handling 👍🏻